Backend/dev/migrations/dynamic-offer-driver-app/0131-added-shardId-in-jobs.sql

```diff
@@ -1 +1 @@
-ALTER TABLE atlas_driver_offer_bpp.allocator_job ADD COLUMN shard_id int;
+ALTER TABLE atlas_driver_offer_bpp.scheduler_job ADD COLUMN shard_id int;
```

Comment:
Why?

Reply:
> Why?

it was rebase issue, in main branch it already changed:

https://github.com/nammayatri/nammayatri/blob/main/Backend/dev/migrations/dynamic-offer-driver-app/0131-added-shardId-in-jobs.sql#L1